### PR TITLE
update 4.12 sdn migration to ovn on aws

### PR DESCRIPTION
**Pull request by zhaozhanqi**

### CODE REVIEW

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml [hunk 1]
```diff
@@ -301,6 +301,17 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-aws-ipi-disconnected-private-sdn-workers-rhcos-rhel8
+- as: e2e-aws-ipi-sdn-migration-ovn
```
Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml:304]: Follow the rules in https://github.com/openshift/release/blob/master/ci-operator/config/openshift/openshift-tests-private/README.md

Comment (zhaozhanqi) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml:304]: `e2e-aws-ipi-sdn-migration-ovn-p2` is ok?

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml [hunk 2]
```diff
@@ -301,6 +301,17 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-aws-ipi-disconnected-private-sdn-workers-rhcos-rhel8
+- as: e2e-aws-ipi-sdn-migration-ovn-p2
+  cron: 0 6 */3 * *
```
Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml:305]: Test frequency need to match the priority

**Pull request merged**